installer/x86_64/install.sh

```diff
@@ -385,6 +385,7 @@ if [ "$install_env" != "sonic" ]; then
     }
 else
     demo_mnt="/host"
+    TAR_EXTRA_OPTION="--numeric-owner"
```

Comment:
Is there a reason to confine this option only to the SONiC-to-SONiC upgrade installation? I believe it should be safe for the ONIE install, as well. In fact, it's probably safer to use it on all installs. I can test an ONIE install using it just to confirm.